icu4c/source/i18n/unicode/calendar.h

```diff
@@ -1349,7 +1349,6 @@ class U_I18N_API Calendar : public UObject {
      */
     virtual UBool isWeekend(void) const;
 
-#ifndef U_HIDE_DRAFT_API
```

Comment:
Don't remove these, rename the switches to U_FORCE_HIDE_DRAFT_API. That's what we use to make the checker tool happy (keeping _HIDE_XXX_API in sync with API tags) when we can't hide the API in real compilation.

Reply:
FYI  https://github.com/unicode-org/icu/blob/main/icu4c/source/test/hdrtst/testtagsguards.sh#L29